webapp/channels/src/components/threading/channel_threads/thread_footer/thread_footer.tsx

```diff
@@ -79,6 +80,10 @@ function ThreadFooter({
         dispatch(setThreadFollow(currentUserId, currentTeamId, threadId, !isFollowing));
     }, [isFollowing]);
 
+    if (post.delete_at > 0 || post.state === Posts.POST_DELETED) {
```

Comment:
Just for my own understanding: what is `post.state`, and why do we need to check both?

Reply:
I'm actually not sure but I noticed both were there and there was another file checking this as well, so I just followed the pattern :rofl: